packages/ui/src/components/ResourceBlock/BaseItem.tsx

```diff
@@ -90,10 +91,15 @@ const BaseItem = ({
         // icon extends margin + icons size: x0.75 + x0.75 = x1.5
         // item margin is base(x1) l-tablet(x1.5)
         data-h2-padding-right="base(x0.5) l-tablet(0)"
+        aria-describedby={descriptionId}
```

Comment:
This should be on the trigger and not the wrapping `div`

Reply:
OK, I think this is better.  Thanks for helping with it!
![image](https://github.com/user-attachments/assets/756a2354-4861-4d7f-938a-723795e3523e)